src/cmd/turnkey/pkg/root.go

```diff
@@ -32,6 +36,9 @@ var (
 	Organization string
 )
 
+// Turnkey Signer enclave's public key.
+const signerPublicKey = "04ca7c0d624c75de6f34af342e87a21e0d8c83efd1bd5b5da0c0177c147f744fba6f01f9f37356f9c617659aafa55f6e0af8d169a8f054d153ab3201901fb63ecb04cf288fe433cc4e1aa0ce1632feac4ea26bf2f5a09dcfe5a42c398e06898710330f0572882f4dbdf0f5304b8fc8703acd69adca9a4bbf7f5d00d20a5e364b2569"
```

Comment:
nit: would be nice to move this to a `keys` directory. You can embed the contents of files directly into the program using [embed](https://pkg.go.dev/embed).

Reply:
hmm we're also thinking of importing this from the go-sdk.. i'll add this to my follow-up PR so we can get import out the door at least.